src/meta/src/hummock/compactor_manager.rs

```diff
@@ -31,11 +34,11 @@ pub type CompactorManagerRef = Arc<CompactorManager>;
 pub struct Compactor {
     context_id: HummockContextId,
     sender: Sender<MetaResult<SubscribeCompactTasksResponse>>,
+    max_concurrent_task_number: u64,
```

Comment:
It's not a good idea to having the number limitation of task  equal to the physical thread number of compactor-node. Because our compact task  always cost most time on IO and during IO request, these tasks are suspend in tokio-scheduler and do not cost any CPU time. If we limit them equal to thread number of tokio, the CPU utilization of compactor would be too low.

Reply:
> It's not a good idea to having the number limitation of task equal to the physical thread number of compactor-node. 

`max_concurrent_task_number` and `compaction_worker_threads_number` can be set separately on demand.

> the CPU utilization of compactor would be too low.

Concurrent task number is a coarse-grained approach to control CPU/memory/network usage. So yes CPU may be underutilized, but it's necessary sometimes, for example if the network quota is the bottleneck, which may lead heartbeat timeout.